package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "payment-sdk",
+  "author": "",
+  "license": "ISC",
+  "version": "1.0.0",
+  "description": "",
+  "homepage": "https://github.com/XcooBee/payment-sdk-js#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/XcooBee/payment-sdk-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/XcooBee/payment-sdk-js/issues"
+  },
+  "main": "dist/node/node.js",
```

Let's add a `files` property.
```suggestion
  "files": ["dist/"],
  "main": "dist/node/node.js",
```
This will ensure that only the stuff we want to publish is in the NPM package. If there is anything else the developer will need to use the package, then update `files` properties.

Note: This is deliberately leaving out `docs/` and `examples/`. NPM packages are bloated enough. The developer can go to GH to see this info.